apps/orchestra/orchestra-rule-unicast-per-neighbor-rpl-storing.c

```diff
@@ -45,6 +45,10 @@
 #include "net/ipv6/uip-ds6-route.h"
 #include "net/packetbuf.h"
 #include "net/rpl/rpl-conf.h"
+#include "net/rpl/rpl-private.h"
+
+#if (RPL_MOP_DEFAULT == RPL_MOP_STORING_NO_MULTICAST) || \
+  (RPL_MOP_DEFAULT == RPL_MOP_STORING_MULTICAST)
```

Comment:
for generality I'd rather check if `UIP_MAX_ROUTES` is 0 or not (when 0, `nbr_routes` is not compiled)